blob/driver/driver.go

```diff
@@ -58,6 +58,8 @@ type WriterOptions struct {
 	// write in a single request, if supported. Larger objects will be split into
 	// multiple requests.
 	BufferSize int
+	// Metadata holds key/value strings to be associated with the blob.
```

Comment:
This doc comment should also specify that the caller is responsible for guaranteeing that all keys are already lowercased.

Reply:
No, the concrete type will lowercase them before creating the driver.WriterOptions. I added a comment to that effect.